readme.md

```diff
@@ -44,7 +44,7 @@ Yields:
 
 ## API
 
-### visit([node](https://github.com/wooorm/unist#unist-nodes)\[, type\], callback\[, reverse\])
+### visit([node](https://github.com/wooorm/unist#unist-nodes)\[, type], callback\[, reverse])
```

Comment:
> Maybe instead of your suggestion we should search for new ways to remove the first escape too?

Well, if we remove the first escape and you ever add a definition for `[, type]`, then it suddenly becomes a reference. We could do a definition lookup but I don't think it would be sustainable.